pytest_fixtures/core/broker.py

```diff
@@ -149,10 +149,12 @@ def capsule_configured(capsule_host, target_sat):
     """Configure the capsule instance with the satellite from settings.server.hostname"""
     capsule_host.capsule_setup(sat_host=target_sat)
     yield capsule_host
+    capsule_host.unregister()
```

Comment:
would it be better to move this step into the capsule_host fixture instead?

Reply:
might be so

Reply:
Althoughtt's a good idea to unregister the host , I think remove-vm WF does that too if I not wrong.